cmd/clusterctl/client/cluster/installer.go

```diff
@@ -35,7 +45,7 @@ type ProviderInstaller interface {
 	Add(repository.Components)
 
 	// Install performs the installation of the providers ready in the install queue.
-	Install() ([]repository.Components, error)
+	Install(InstallOptions) ([]repository.Components, error)
```

Comment:
this is technically a breaking change from a clusterctl library perspective. @fabriziopandini should the commit have the ⚠️ prefix?

Reply:
We are not currently providing any compatibility guarantees on cluserctl used as a library, but I agree we should put a warning on this change, good catch.